CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.1.125-dev
+
+* fixed false positives in `prefer_colleciton_literals`.
```

Comment:
```suggestion
* Fixed false positives in `prefer_colleciton_literals`.
```

Reply:
Trying to match style with the rest of the file.